setup/teardown.js

```diff
@@ -0,0 +1,15 @@
+// teardown.js
+const print = require('./print');
+const os = require('os');
+const rimraf = require('rimraf');
+const path = require('path');
+
+const DIR = path.join(os.tmpdir(), 'jest_puppeteer_global_setup');
+module.exports = async function() {
+  // close the browser instance
+  print('Teardown Puppeteer','green');
+  await global.__BROWSER_GLOBAL__.close();
+
+  // clean-up the wsEndpoint file
```

Comment:
What is wsEndpoint?

Reply:
**browserWSEndpoint** <?string> a browser websocket endpoint to connect to.
A browser url to connect to, in format http://${host}:${port}. Use interchangeably with browserWSEndpoint to let Puppeteer fetch it from metadata endpoint.

> clean-up the wsEndpoint file
Basicly its end instance.